generator.js

```diff
@@ -43,11 +44,21 @@ const processFile = (file, currentPath) => {
 
 const filePath = path.join(__dirname, 'template')
 const projectPath = path.join(process.cwd(), appTitle)
+
 console.log(`Creating ${projectPath} `)
 console.log('Copying over project files')
 processFile('', filePath)
 console.log('Installing NPM Depenedencies')
 execSync('npm install', {cwd: projectPath, stdio: 'inherit'})
+console.log('Initializing a git repository')
+console.log('Making the initial commit !')
+const simplegit = git(projectPath)
+simplegit.init().add('./*').commit('Init Tram-One-App !', (e, d) => {
```

Comment:
* not `add('.')`? 
* if we could make this string instead `'Initial commit from Tram-One Express'`, this follows the commit message that create-react-app uses
* please use full variables here, not `e` and `d` (we don't have to worry about bundle size here 😄)